kotlin-dsl/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.2.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.1.1-bin.zip
```

Comment:
Why decreasing the version ?

Reply:
I checked [there](https://kotlinlang.org/docs/gradle-configure-project.html#apply-the-plugin).
See the first table.